### ISSUE TITLE
Trucks stopping while delivering to storage!

### ISSUE THREAD

**Issue by ManBo39** [MaFi-Games/Captain-of-Industry-issues#542]

**Comment by ParanoikCZE** [comment]
Sorry, your description is not accurate to understand your problem, but this is possibly duplicate issue of #474.

**Comment by ManBo39** [comment]
I did mention this earlier in #445 but that one was closed, so i do mention it again!

**Comment by ParanoikCZE** [comment]
But that item is about something else. So, I didn't check your saves, do you have issues with depot or just stopping trucks while something is deploying to their destination?

**Comment by ManBo39** [comment]
I did mention it in my previous issue ( https://github.com/MaFi-Games/Captain-of-Industry-issues/issues/445#issuecomment-1171473353 ), I also communicated with Marek about this on discord (read my latest posts on discord). If a truck is going to deliver a material and there are several trucks assigned to this storage (where the material comes from) the other trucks will stop as soon as the first truck makes a delivery, when the first truck is finished the other truck move on until te second truck makes a delivery, then the trucks stops again. Marek suggested I could use small storage and it wouldn't happen, but no matter what storage I drop the problem is still there.
